packages/postcss-lowercase-text/src/__tests__/atRules.test.js

```diff
@@ -0,0 +1,120 @@
+import postcss from 'postcss';
+
+import plugin from '..';
+
+function run(testMsg = 'should work', input, output) {
+  it(testMsg, (done) => {
+    let result = postcss([plugin]).process(input, {
+      from: undefined,
+    });
+    expect(result.css).toEqual(output);
+    done();
+  });
+}
+
+describe('transforming @media', () => {
+  run(
+    'should safely transform the @media and its properties',
+    '@MEDIA screen and (min-width: 480px){body{COLOR: lightgreen;}}',
+    '@media screen and (min-width: 480px){body{color: lightgreen;}}'
+  );
+
+  run(
+    'should safely transform the @media but not the params',
+    '@MEDIA SCREEN and (min-width: 480px){}',
+    '@media screen and (min-width: 480px){}'
+  );
+});
+
+describe('transforming @charset', () => {
+  run(
+    'should not transform the @charset',
+    '@charset "UTF-8";',
+    '@charset "UTF-8";'
+  );
+  run(
+    'should safely transform the @charset',
+    '@CHARSET "iso-8859-15";',
+    '@charset "iso-8859-15";'
+  );
+});
+
+describe('transforming @import ', () => {
+  run(
+    'should  transform the @import ',
+    `@IMPORT url("fineprint.css") print;@import url("bluish.css") speech;@ImPoRt 'custom.css';@import url("CHROME://communicator/SKIN/");@import "common.css" screen;`,
+    `@import url("fineprint.css") print;@import url("bluish.css") speech;@import 'custom.css';@import url("CHROME://communicator/SKIN/");@import "common.css" screen;`
+  );
+});
+
+describe('transforming @namespace', () => {
+  run(
+    'should safely transform the @namespace',
+    '@NAMESPACE prefix url(http://www.w3.org/1999/xhtml);',
+    '@namespace prefix url(http://www.w3.org/1999/xhtml);'
```

Comment:
Please add test `@NAMESPACE SVG URL(http://www.w3.org/2000/svg);`, `SVG` should be in uppercase (keep as is)